### PR TITLE
feat: enable 'Remove liquidity' flow when unlockPeriod = 0

### PR DESCRIPTION
**Pull request by alanrsoares**
* Enable 'Remove liquidity' ui flow when `rewardsParams.liquidityRemovalLockPeriod` is set to `0`

### CODE REVIEW

**Comment by alanrsoares** [comment]
@pgoos @sheokapr to clarify this PR, this is not using the `unbond liquidity` tx at all. The UI falls back entirely to the old `remove liquidity` flow.

**Comment by sheokapr** [comment]
@alanrsoares -- I can see 2 issues here: 
1) Slider on Remove-Liquidity Modal - "Withdraw Amount" is not working with keys, percentage number is not changing however numbers in 'You will Receive" section move.

image

image


**2) Getting 501 error when trying to 'remove-liquidity'**

image

**Comment by alanrsoares** [comment]
@sheokapr first issue solved, the slider step was too small, you'd have to keep the arrow key pressed over 3s to notice it was moving. Should be more responsive now

**Comment by alanrsoares** [comment]
@sheokapr @pgoos tried testing this on devnet, that error you're getting is because of the .45 upgrade. However, after fixing that locally I started getting other errors: 
* "Cannot remove liquidity asynchronously" when trying to change the second range input
* "Insuficient balance" when only changing the first range input (due to sifnode still requiring liquidity unbonding)

**Comment by alanrsoares** [comment]
@sheokapr @pgoos this is working now when  pointing to mainnet

**Comment by alanrsoares** [comment]
@pgoos @sheokapr PR is updated with cosmos sdk .45 support

**Comment by sheokapr** [comment]
I can see multiple issues (used this vercel-build for testing this -- https://sifchain-p022he2sj-sifchain.vercel.app/  ) (edited) 

1. Unbonding period is set to 10 (not 0) in TestNet, FE is showing 'Remove-Liquidity' button on FE instead of unbond-liquidity'
(edited)

image

2. When open Remove-Liquidity modal, it doesn't show data (could be a new testnet backend issue), also slider is not working after first setting (edited) 


image

**Comment by pgoos** [comment]
@alanrsoares I allowed myself to fix the conflicts on your branch ;)

File: app/src/business/services/ClpService/ClpService.ts [hunk 1]
```diff
@@ -149,25 +149,30 @@ export class ClpService {
...
     await this.tokenRegistry.load();
     const entry = await this.tokenRegistry.findAssetEntryOrThrow(params.asset);
-    const response = await this.client.getLiquidityProvider({
-      // cannot use params.asset.ibcDenom because ibcDenom is set when loading balances,
-      // and the user does not always have a balance for the asset they have pooled
-      symbol: entry.denom,
-      lpAddress: params.lpAddress,
-    });
 
-    const { liquidity_provider, native_asset_balance, external_asset_balance } =
```
Comment (tien) [inline app/src/business/services/ClpService/ClpService.ts:159]: 🚫🐍

**Review by tien — APPROVED** [review]

**Pull request merged**